metrics/wer/README.md

```diff
@@ -0,0 +1,118 @@
+# Metric Card for WER
+
+## Metric description
+Word error rate (WER) is a common metric of the performance of an automatic speech recognition (ASR) system. 
+
+The general difficulty of measuring performance lies in the fact that the recognized word sequence can have a different length from the reference word sequence (supposedly the correct one). The WER is derived from the [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance), working at the word level instead of the phoneme level. 
+
+This problem is solved by first aligning the recognized word sequence with the reference (spoken) word sequence using dynamic string alignment. Examination of this issue is seen through a theory called the power law that states the correlation between [perplexity](https://huggingface.co/metrics/perplexity) and word error rate.
+
+Word error rate can then be computed as:
+
+`WER = (S + D + I) / N = (S + D + I) / (S + D + C)`
+
+where
+
+`S` is the number of substitutions,
+
+`D` is the number of deletions,
+
+`I` is the number of insertions,
+
+`C` is the number of correct words,
+
+`N` is the number of words in the reference (`N=S+D+C`).
+
+
+## How to use 
+
+The metric takes two inputs: references (a list of references for each speech input) and predictions (a list of transcriptions to score).
+
+
+```python
+from datasets import load_metric
+wer = load_metric("wer")
+wer_score = wer.compute(predictions=predictions, references=references)
+```
+## Output values
+
+This metric outputs a float representing the word error rate.
+
+```
+print(wer_score)
+0.5
+```
+
+This value indicates the average number of errors per reference word. 
+
+The **lower** the value, the **better** the performance of the ASR system, with a WER of 0 being a perfect score.
+
+## Values from popular papers
```

Comment:
I think I would actually delete this section as it's not very relevant to the definition of WER IMO. WER's are always relative, therefore I'm not sure how meaningful absolute comparisons of WER values are. Also common datasets and "recent" systems are always changing so I think it would be very difficult to keep this section up to date

Reply:
This is part of the template for metric cards (and we are trying to keep that standard across metrics). 
I tried to find some historical/popular values, and I don't think we need to keep this updated, it's just to give people an idea of the evolution of WER